tools/README.md

```diff
@@ -112,115 +77,199 @@ export repo_owner=<your github account>
+
+We outline below the rapid startup steps to get SonarQube Server setup. Refer to the original repository for more details if necessary.
+
+#### SonarQube Deploy
+
+You will need to use the `sonarqube-postgresql-template.yaml` file in order to deploy it to openshift. This will deploy BOTH the PostgreSQL database AND the SonarQube server.
+
+*Note: At the time of writing, the master branch of BCDevOps SonarQube repo was at commit `bbb9f62e29706b61382cf24d7ad7e08f2476a01f`.  And the template was at commit `bc80961d75eed66ec70ca022a6444963341fb39f`.*
+
+Deploying the database is done with the following:
 
 ```sh
-oc project $tools
+oc -n $tools new-app -f https://raw.githubusercontent.com/BCDevOps/sonarqube/bc80961d75eed66ec70ca022a6444963341fb39f/sonarqube-postgresql-template.yaml --param=SONARQUBE_VERSION=6.7.5
 ```
 
-### Create Secrets
-The BCDevOps CICD Jenkins Basic install requires a template github secret and a template for the slave.  This will create the secrets named as it requires.
+Let the SonarQube pods spin up, you can then go to `https://sonarqube-<$tools>.pathfinder.gov.bc.ca` and watch until SonarQube is ready.  Now you can set the admin password.
+
+#### Admin Password
+
+The SonarQube server instance is created with standard insecure credentials (User: `admin`, PW:). This should be reset to something stronger and stored in an OpenShift secret so authorized developers can find it.
+
+The BCDevOps SonarQube repo provides a script that will generate a random PW, set it in SonarQube, and create a secret. This can be found under the  `/provisioning` folder of the cloned BCDevOps repo.
+
+In order to directly get the password reset script, run the following:
 
 ```sh
-oc -n $tools process -f "$templates_url/secrets.yaml" -p GH_USERNAME=$gh_username -p GH_PASSWORD=$gh_password | oc  -n $tools create -f -
+curl https://raw.githubusercontent.com/BCDevOps/sonarqube/bbb9f62e29706b61382cf24d7ad7e08f2476a01f/provisioning/updatesqadminpw.sh > updatesqadminpw.sh
 ```
 
-### Create ConfigMap for related namespaces
-For our custom jobs scripts and Jenkinsfiles.
+Then simply run the following script and follow its instructions. Make sure you save the new password in an OpenShift secret or equivalent!  Ensure that the SonarQube app is fully deployed and operational before running this script.
 
 ```sh
-oc -n $tools process -f "$templates_url/ns-config.yaml" -p DEV=$dev -p TEST=$test -p PROD=$prod -p TOOLS=$tools | oc  -n $tools create -f -
+chmod +x updatesqadminpw.sh && ./updatesqadminpw.sh
 ```
 
-### Create ConfigMap for the application
-For our custom jobs scripts and Jenkinsfiles.
+Go to `https://sonarqube-<$tools>.pathfinder.gov.bc.ca` and log in as `admin` with the new password (it is stored in sonarqube-admin-password secret).
+
+### SonarQube Scanner
```

Comment:
I had this section at the end because it's a little confusing - it really has nothing to do with the tools repo but more to do with local development.  Can we move this section to the bottom with a link "For local development" or something like that?  Setting up (and having the steps for installing) SOnarQube first is good.

Reply:
I moved that chunk down to the very bottom and made a top level section called "Local Development Notes" to encapsulate it. It does feel a bit off having this stuff not be with the sonarqube content though - but should be good enough for now.

Reply:
It's about setting up the tools namespace, which that section isn't really part of - but it's good information to share - just not relevant to the task at hand.